aardwolf-server/Cargo.toml

```diff
@@ -29,8 +30,8 @@ rocket = "=0.3.8"
 rocket_codegen = "=0.3.8"
 serde = "1.0.21"
 serde_derive = "1.0.21"
-config = "0.8.0"
-clap = {version = "2.29", features = ["yaml"]}
```

Comment:
Why was CLAP removed? It is used by aardwolf-server/src/bin/main.rs and aardwolf-server/src/bin/setup.rs to interpret the command-line options.

Reply:
it was not removed, just moved lower in the file

Reply:
D'oh! I even red that part of the file like 5 times, and it never clicked... LOL